src/main/java/org/paymentsds/mpesa/Request.java

```diff
@@ -72,7 +72,11 @@ public Builder amount(double amount) {
         }
 
         public Builder from(String from) {
-            this.from = from;
+            if (from.length() == 9) {
```

Comment:
Checking the string's length sounds a bit hacky. Maybe we should check if the string starts with "258" instead?